test/framework/twilio-video-react/src/index.js

```diff
@@ -19,8 +19,17 @@ function getQueryParameters(location) {
 
 // eslint-disable-next-line
 const token = (getQueryParameters(location).get('token') || [])[0] || '';
+// eslint-disable-next-line
+const environment = (getQueryParameters(location).get('environment') || [])[0] || '';
```

Comment:
by ensuring that you always have correct `environment` you can avoid code changes in react render block below.
```suggestion
const environment = (getQueryParameters(location).get('environment') || [])[0] || 'prod';
```